commands/mr/create/mr_create.go

```diff
@@ -553,6 +553,10 @@ func generateMRCompareURL(opts *CreateOpts) (string, error) {
 		// this uses the slash commands to add milestone to the description
 		description += fmt.Sprintf("\n/milestone %%%d", opts.MileStone)
 	}
+	if opts.IsWIP || opts.IsDraft {
+		// this uses quick-actions to make the merge request a work in progress
+		description += "\n/wip"
+	}
```

Comment:
Some users prefer to use `DRAFT:` and some prefer `WIP:`. 
`glab` provides both flags which are mutually exclusive
```suggestion
	if opts.IsWIP || opts.IsDraft {
		// this uses quick-actions to make the merge request a work in progress
		description += "\n/wip"
	} else if opts.IsDraft {
		// this uses quick-actions to make the merge request as draft
		description += "\n/draft"
	}
```